### DISCUSSION TITLE
Master key location

### DISCUSSION THREAD

**Discussion by Anushka3174** [decentralized-identity/veramo#425] [Q&A]
Where is the private key/seed stored locally?

**Comment by jasheal** [comment]
Veramo is plugin based so you can save the private key anywhere. The default config uses `kms-local` which stores it encrypted in the database. https://github.com/uport-project/veramo/tree/next/packages/kms-local

This library allows you to use AWS
https://www.npmjs.com/package/@sertoid/veramo-aws-kms
